website/docs/migrations/12-13.md

```diff
@@ -0,0 +1,792 @@
+
+```jsx
+<JsxSvg svg="<svg ..." />
+```
+
+## Styles
+
+### Utility classes
+Frontend libs utility classes are deprecated and will be removed in the next version.
+
+Remove them from your code if possible, and check if some of them can be replaced with ones from ES UI components (they have the `es-uic-` prefix).
+
+### CSS reset
+The default CSS reset was updated. Do a quick look-around on your site to see if anything is different, but nothing major is expected.
+
+### SCSS
+Some SCSS functionalities were removed from Frontend libs.
+
+If you were using them, you can re-define them in your project. The previous implementation can be checked in Frontend libs v12 source.
+
+#### Placeholders
+Removed placeholders:
+- `%absolute-center`
+- `%bg-image`
+- `%clearfix`
+- `%site-padding`
+- `%site-padding-correction`
+- `%link-transition`
+- `%heading-reset`
+- `%paragraph-reset`
+- `%dl-reset`
+
+#### Functions
+Removed functions:
+- `calc-column-width` / `calcCW`
+- `calc-dynamic-size` / `calcDS`
+- `calc-grid-width` / `calcGW`
+- `line-height` / `lh`
+- `em`
+
+#### Keyframes
+Keyframes were removed from Frontend libs
+
+#### Mixins
+Removed mixins:
+- `flex-container`
+- `box-sizing` (replace with `box-sizing`, if no arguments were specified, the default is `border-box`)
+- `column-offset-modifiers`
+- `flex-column-width-modifiers`
+- `flex-column-width`
+- `flex-grid-width-modifiers`
+- `flex-grid-width`
+- `flex-horizontal-align-modifiers`
+- `flex-vertical-align-modifiers`
+- `text-align-modifiers`
+- `stretch`
+- `placeholder`
+- `inline-font-colors`
+- `grid-offset-modifiers`
+- `font-smoothing`
+- `custom-bullets`
+- `link-modifiers`
+- `link`
+- `modifiers-deep`
+- `modifiers-range`
+- `modifiers`
+- `responsive-modifiers`
+- `responsive-selectors-visibility`
+- `responsive-selectors`
+- `responsive`
+- `for-each-attribute`
+- `es-eases`
+
+### Component styles, editor overrides, ...
+The previously included Gutenberg component style overrides are not included by default anymore. `overrides-editor.scss` is still present, but only a couple of styles are left for compatibility reasons. Instead, you can `@import '~@eightshift/ui-components/dist/assets/wp-ui-enhancements.css';` from Eightshift UI components.
+
+`es-component-styles.scss` (in Frontend libs) are not bundled with `overrides-editor.scss` anymore, and should be imported separately if needed.
+
+## Replace components
+
+This will take some time, but the guide below should make it easier. Our recommendation is to go block by block (instead of replacing one type of UI component at a time).
+
+:::note
+Some components will still have fallbacks not to break everything immediately, but replacing them is **strongly** recommended, as everything will be removed in the next version.
+:::
+
+:::tip
+After replacing components, check if there are any new props that might be useful for your usage.
```

Comment:
```suggestion
After replacing components, check if there are any new props that might be useful for your situation.
```
Or 
```suggestion
After replacing components, check if there are any new props that might be useful for your project.
```

Reply:
I'll go with _with your use case_, but thanks for the idea 😄